Library/Contributions/example-formula.rb

```diff
@@ -165,7 +168,7 @@ def pour_bottle?
   depends_on :hg # Mercurial (external or brewed) is needed
 
   # If any Python >= 2.7 < 3.x is okay (either from OS X or brewed):
-  depends_on :python
+  depends_on :python if MacOS.version <= :snow_leopard
```

Comment:
Wouldn't ` depends_on :python ` and ` depends_on :python if MacOS.version <= :snow_leopard ` achieve the same thing on OS X >10.6?

I'm inevitably wrong on that, so I'll fix it, but fear I may have misunderstood the way we handle ` :python ` around this conditional.